client/js/middleware/serializers/qbank/multiple_choice_serializer.js

```diff
@@ -0,0 +1,70 @@
+import _              from 'lodash';
```

Comment:
We should probably add a comment somewhere (not sure where, maybe here or in the reducer) showing what the end object looks like.